jujugui/static/gui/src/app/components/spinner/spinner.js

```diff
@@ -18,7 +18,7 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 'use strict';
 
-class Spinner extends React.Component {
+class Spinner extends React.PureComponent {
```

Comment:
Possibly, but why in this branch? Also, that would require an update on the isnpect-component script.